### PR TITLE
Third try on annotations dashboard ticks

### PR DESCRIPTION
**Pull request by codyml**
...this time by explicitly setting the tick values.  Hope this one works?

### CODE REVIEW

**Comment by mwidner** [comment]
@codyml I'm still seeing a bunch of extra space to the left-hand side of the time filter. 
screen shot 2016-11-29 at 11 02 34 am

**Comment by codyml** [comment]
That should've fixed it!

screen shot 2016-11-30 at 9 43 17 pm

File: sites/all/modules/custom/lacuna_visualizations/libraries/d3.annotations_dashboard/annotations_dashboard.js [hunk 1]
```diff
@@ -886,7 +886,7 @@ function main(data) {
 
   	bars.attr("width", bar_width)
   		.attr("y", function (d) { return bar_y(d.y) })
-  		.attr("x", function (d) { return bar_x(d.x) })
+  		.attr("x", function (d) { console.log(bar_x(d.x)); return bar_x(d.x) })
```
Comment (codyml) [inline sites/all/modules/custom/lacuna_visualizations/libraries/d3.annotations_dashboard/annotations_dashboard.js]: D: these ticks have me losing my mind!!!

Comment (codyml) [inline sites/all/modules/custom/lacuna_visualizations/libraries/d3.annotations_dashboard/annotations_dashboard.js]: (sorry)

**Pull request merged**